api/proto/ipc/tournament.proto

```diff
@@ -112,6 +112,11 @@ message RoundControl {
   int32 repeat_relative_weight = 9;
   int32 win_difference_relative_weight = 10;
   bool interleave_team_round_robin = 11;
+  int32 spread_cap_override = 12;
+  // We use this boolean to differentiate between a disabled override
+  // (i.e. using the division default) and an enabled override set to 0
+  // (i.e. overriding the division default to remove the cap)
+  bool spread_cap_override_enabled = 13;
```

Comment:
Without optionals, `0` could potentially represent both:
 - Override the tournament-level cap and set the cap to `0` (i.e. remove the cap if it exists)
 - No round-override

Other options here are:
 - making `spread_cap_override` optional. Opted against this since code search shows no other optional fields in the entire project
 - allowing negatives, where `-1` means no override
 - Disallow either: setting a spread cap override when no tournament-level cap is set _or_ having an override which removes the spread-cap, therefore making `0` only represent one possible situation

I think we should either allow optionals in this instance or keep this boolean + int set-up, the other two are unappealing to me. But wanted to flag all the options

Reply:
I don't have to be super dogmatic about optionals; it does make sense in this case to have optionals. I'll say I would also be ok with a spread cap of 0 meaning no spread cap, since it seems we might never want a spread cap of 0 (what would it mean? every game is a binary win/loss? I guess it makes some sense).

I do prefer an optional over having two variables for it. 

Looking at this now:

https://protobuf.dev/programming-guides/field_presence/

It seems that the protobuf people changed their minds again about explicit vs implicit presence. So now they recommend `optional` again (it wasn't in proto3 for the longest time, then experimental, and now they recommend it?).